src/cloud-api-adaptor/pkg/forwarder/forwarder.go

```diff
@@ -44,8 +44,8 @@ type Config struct {
 	TLSClientCA   string `json:"tls-client-ca,omitempty"`
 
 	AAKBCParams string `json:"aa-kbc-params,omitempty"`
-
-	AuthJson string `json:"auth-json,omitempty"`
+	KBSCERT     string `json:"kbs-cert,omitempty"`
```

Comment:
I don't think we need to add this to daemon.json (we should also drop AAKBCParams, it's unused afaik)